csvs/tasks.csv

```diff
@@ -74,3 +74,13 @@ Website,"no more stickies on website",357,358,angle943,11/1/20,500
 Website,"Footer Update V2",330,359,angle943,11/1/20,500
 Website,"Put "Hello World" in your OWN language!",351,361,aayushi14,11/1/20,500
 Website,"Put "Hello World" in your OWN language!",351,363,moki298,11/1/20,500
+Website,"Add a Q&A on how to open up a pull request",353,367,aayushi14,11/1/20,1000
+Website,"When going to `/tasks`, make it automatically default to `/tasks/All`",349,365,aayushi14,11/1/20,2000
+Website,"Make the Footer's background color go all the way to the bottom",369,370,itsnikhil,11/2/20,1000
+Website,"Put "Hello World" in your OWN language!",369,351,itsnikhil,11/2/20,500
+Website,"Put "Hello World" in your OWN language!",369,380,xmac11,11/2/20,500
+Website,"Put "Hello World" in your OWN language!",369,381,rizqyep,11/2/20,500
+Website,"Put "Hello World" in your OWN language!",369,382,DucPhamTV,11/2/20,500
+Website,"Put "Hello World" in your OWN language!",369,379,shahraizali,11/2/20,500
+Account-Manager,"While Editing nickname duplicate name should not be accepted",shahraizali,400,397,10/13/20,2500
+thenewboston-python,"JSONDecodeError simplejson.decoder in raw_decode",shahraizali,,22,10/16/20,5000
```

Comment:
@buckyroberts please advise on this. This PR was open on `thenewboston-python`, but the issue is on `Validator`

Reply:
@angle943 It will be where the Issue was opened since that's how we link to it on the FE.

<img width="688" alt="Screen Shot 2020-11-02 at 7 36 03 PM" src="https://user-
images.githubusercontent.com/8547538/97947613-ac423800-1d42-11eb-85f6-17d144d8de78.png">